mirth.rb

```diff
@@ -17,8 +18,10 @@ def all_paths
   end
 
   def show_data
-    template = <<~HERE
-      <ul><% DailyData.all.each do |daily_data| %> 
+    all_daily_data = DailyData.all
+
+    template = ERB.new <<~HERE
```

Comment:
This is clever but you can let Action Controller do the work for you: just assign `DailyData.all` to an instance variable, and it will automatically make the controller’s instance variables available inside the template so you can refer to it there.

Reply:
(So you don’t need to use `ERB.new` or `.binding` at all, just a regular string.)

Reply:
Oh!!! Interesting. I didn't know that. More magic from action controller/view. 

(I got the `#binding` idea from one of the linked resources.)